lib/ui/addon/components/toolbar-search-field.hbs

```diff
@@ -1,15 +1,17 @@
-<div class="toolbar__search" ...attributes>
+<div class="toolbar__search relative mx-1" ...attributes>
   <Input
     @id={{this.inputId}}
     @type="text"
     @value={{@value}}
     placeholder="Search"
+    class="box-border m-0 rounded pt-0 pr-4 pb-0 pl-1 text-12 leading-none outline-none bg-base00"
   />
 
   {{#if @value}}
     <button
-      class="toolbar__icon-button toolbar__search-clear-button js-search-field-clear-button"
+      class="toolbar__icon-button toolbar__search-clear-button absolute right-0 w-auto h-auto"
       title="clear"
+      data-test="search-field-clear-button"
```

Comment:
Could we use https://github.com/simplabs/ember-test-selectors for these selectors?

```suggestion
      data-test-search-field-clear-button
```

Reply:
That's cool with me. I started down this road because it felt odd to have `js-test-xyz` along with a bunch of utility classes. If that seems like a good path I will add ember-test-selectors.

Reply:
Yeah, I am a big fan of ember-test-selectors 👍

Reply:
@rwwagner90, upgraded ember-test-selectors and moved everything in the app with `data-test="*"` to `data-test-*`